src/sentry/api/serializers/models/sentry_app_component.py

```diff
@@ -18,14 +18,16 @@ def serialize(self, obj, attrs, user):
 
 
 class SentryAppAlertRuleActionSerializer(Serializer):
-    def serialize(self, obj, attrs, user, install, **kwargs):
+    def serialize(self, obj, attrs, user, **kwargs):
+        install = kwargs.get("install")
+        event_action = kwargs.get("event_action")
```

Comment:
You'll need a default value for `install` and `event_action` in case someone tries using this serializer without passing them in.

Reply:
These should be provided in order for the serializer to be useful, I will instead throw an error if it is called without specifying these values